showcase.json

```diff
@@ -907,6 +907,220 @@
       "/img/showcase/astrox_me_screenshot.webp"
     ]
   },
+  {
+    "id": "funded",
+    "name": "Funded",
+    "onLiner": "Crowdfund the Future! Using smart contract technology to build a safer, cheaper and global crowdfunding service - with a new NFT model.",
+    "tags": [
+      "SocialFi",
+      "Tools / Infrastructure"
+    ],
+    "useInternetIdentity": true,
+    "website": "https://funded.app/",
+    "github": "https://github.com/CrowdFund-NFT/escrow_manager",
+    "twitter": "https://twitter.com/ApocStudiosInc?s=20&t=jqHbKZjRS47NdOcbDHfK6w",
+    "discord": "https://discord.gg/thqARE5JqF",
+    "logo": "/img/showcase/funded_logo.webp",
+     "submittableId": "26383175"
+  },
+  {
+    "id": "argonstudio",
+    "name": "Argon Studio",
+    "onLiner": "We'll be your end-to-end partner, whether managing your project from ideation to implementation, or filling gaps in your current team.",
+    "tags": [
+      "Tools / Infrastructure"
+    ],
+    "website": "https://argonstudios.xyz/",
+    "twitter": "https://twitter.com/ArgonStudiosXYZ",
+    "logo": "/img/showcase/argonstudio_logo.webp",
+     "submittableId": "34524251"
+  },
+  {
+    "id": "aviatelabs",
+    "name": "Aviate Labs",
+    "onLiner": "Stop worrying about system failures and slow performance. Our datacenter standby team is here to keep your nodes running at top speed, with round-the-clock monitoring and troubleshooting. Currently managing 70 nodes in 3 datacenters.",
```

Comment:
This one doesn't display either - I think something more fundamental broke: 
<img width="1366" alt="image" src="https://user-images.githubusercontent.com/14757975/229052830-fb775311-740f-4dd8-9653-6bfe273608fe.png">